stable/datacube-dev-env/values.yaml

```diff
@@ -0,0 +1,88 @@
+# Default values for datacube-wms.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+environment: dev
+image:
+  registry: docker.io
+  repository: opendatacube/k8s-dev
+  tag: "latest"
+  pullPolicy: Always
+
+datacube:
+  configPath: /opt/odc/datacube.conf
+
+database:
+  create: false
+  database: testdb
+  ref_database: ows
+  host: database.local
+  port: 5432
+  adminSecret: dev-eks-datacube
+  existingSecret: dev-eks-datacube
+
+clusterInfo:
+  envName: dev
+
+resources:
+  limits:
+    cpu: "300m"
+    memory: 1024Mi
+
+dockerArgs: [ "/bin/bash", "-c", "tail -f /dev/null" ]
+
+annotations:
+  iam.amazonaws.com/role: dev-eks-datacube-wms
+
+service:
+  type: NodePort
+  port: 80
+
+createdb:
+  dockerArgs: [ "/bin/sh", "-c", "/code/setup/create-db.sh" ]
+  image:
+    registry: docker.io
+    repository: opendatacube/wms
+    tag: latest
+    pullPolicy: Always
+
+setup:
+  dockerArgs:
+    - "/bin/bash"
+    - "-c"
+    - "echo 'CREATE EXTENSION IF NOT EXISTS postgis; \n grant agdc_admin to $ADMIN_USERNAME;' |
+       PGPASSWORD=$ADMIN_PASSWORD psql -h $DB_HOSTNAME -p 5432 -U $ADMIN_USERNAME -d $DB_DATABASE"
+  image:
+    registry: docker.io
+    repository: postgres
+    tag: latest
+    pullPolicy: Always
```

Comment:
This should be removed with the postgis setup

Reply:
Removed postgis setup from values.yaml file